app/views/jobseekers/job_applications/review/_employment_history.html.slim

```diff
@@ -33,7 +33,7 @@
             - row.with_key text: t("helpers.legend.jobseekers_job_application_details_employment_form.is_current_role")
             - row.with_value text: employment.is_current_role ? "Yes" : "No"
 
-          - unless employment.is_current_role
+          - unless employment.current_role?
```

Comment:
:raising_hand: Is this intended? Didn't we move away from `current_role?` in favour of the new boolean field?